oxcaml/tests/backend/validators/check_regalloc_validation.ml

```diff
@@ -88,7 +89,7 @@ let base_templ () : Cfg_desc.t * (unit -> InstructionId.t) =
      [fun f x y a -> let y = f x y a in let x = x + y in x] *)
   let int_arg1 = int.(0) in
   let int_arg2 = int.(1) in
-  let stack_loc = Reg.create_at_location int_arg1.typ (Stack (Local 0)) in
+  let stack_loc = Reg.For_testing.with_loc int_arg1 (Stack (Local 0)) in
```

Comment:
(That does not really matter in this
context, and I think the new version
is slightly better, but contrary to most
of the other changes, this one is not
strictly equivalent - for instance a
name would be propagated.)

Reply:
it was intentional, I thought the new version is better. sorry I forgot to mention it in the description.